### PR TITLE
MTL-1501 Fail to Disk

### PR DESCRIPTION
**Pull request by rustydb**
#### Summary and Scope

- Fixes MTL-1501
- Relates to MTL-1572

##### Issue Type

- Bugfix Pull Request

This change adds new logging and earlier exits to the metal iPXE configuration. The new early exits will print the error cause, wait for 5 seconds, and then bail. The bail exits the ROM and causes the BIOS to move onto the next boot option.

#### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
- [x] I tested this on internal system (x) (if yes, please include results or a description of the test)
 

#### Risks and Mitigations
 
Some servers may fall to BIOS if their boot order is not configured, or if they have a feature-toggle on for exit on failure (rare).

This is low risk, but worth noting.

### CODE REVIEW

File: .github/workflows/merge-strategy.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Merge Strategy
```
Comment (heemstra) [inline .github/workflows/merge-strategy.yml:1]: This should be a separate PR. You wouldn't want it caught up in a revert.

**Review by heemstra — CHANGES_REQUESTED** [review]

File: .github/workflows/merge-strategy.yml (hunk 1 above)
Comment (rustydb) [reply .github/workflows/merge-strategy.yml:1]: Thanks for noticing. For what it is worth I actually didn't mean to include this. I created this branch off my local block-merge-commits branch and didn't pay attention to the overzealous include.

Will factor this out.

**Comment by rustydb** [comment]
I'm not sure why but the cmdline isn't printing anymore, I stared at the diff for 5minutes and couldn't figure out what happened to change it. Perhaps fresh eyes in the AM will figure it out:
```bash
64 bytes from 10.1.1.2: seq=1
Successfully called home; deployment server is reachable
http://pit/ncn-w003/initrd.img.xz... ok
initrd.img.xz : 52780376 bytes
http://pit/ncn-w003/kernel... ok

initramFS and kernel succesfully downloaded.

Starting up...
```
(cmdline should dump before `Starting up...` is printed)

File: .github/workflows/merge-strategy.yml (hunk 1 above)
Comment (rustydb) [reply .github/workflows/merge-strategy.yml]: (for deeper context, this file is already in `main` and I didn't rebase on the latest main when I pushed this)

**Review by heemstra — APPROVED** [review]

**Pull request merged**